### ISSUE TITLE
REQUEST: netmaker

### ISSUE THREAD

**Issue by cmonty14** [mimugmail/opn-repo#112]
Open source wireguard mesh network client.
https://github.com/gravitl/netmaker

**Comment by mimugmail** [comment]
Hm, this would need a FreeBSD port, will take some time

**Comment by cmonty14** [comment]
The package is available [here](https://github.com/gravitl/netmaker/releases/tag/v0.9.4).

**Comment by mimugmail** [comment]
Yep, but this is a package, not a port

**Comment by cmonty14** [comment]
> Yep, but this is a package, not a port

Right, actually it's the binary w/o any config, service file, etc.

**Comment by mimugmail** [comment]
You can try to grab the binary, test the setup and write a small example guide, would save me some testing

**Comment by cmonty14** [comment]
Sure...
Unfortunately I didn't complete the setup of Netmaker server, yet.

However, the netmaker-client (= netclient) binary seems to be very intuitive:
```
$ netclient --help
2022/06/12 13:16:22 This program must be run with elevated privileges (sudo). This program installs a SystemD service and configures WireGuard and networking rules. Please re-run with sudo/root.
```

I was instructed to run `netclient join`; this will install and start the daemon, create config files in/etc/netclient and log file in /var/log.

**Comment by mimugmail** [comment]
But so you want the client or server in OPN?

**Comment by cmonty14** [comment]
> But so you want the client or server in OPN?

Client only.

(Server will run on dedicated instance, VM or K3s.)
